ribs/emitters/operators/_iso_line.py

```diff
@@ -0,0 +1,57 @@
+"""Iso Line Operator"""
+import numpy as np
+
+from ribs.emitters.operators._operator_base import OperatorBase
+
+
+class IsoLineOperator(OperatorBase):
+    """Args:
+        archive (ribs.archives.ArchiveBase): An archive to use when creating and
+            inserting solutions. For instance, this can be
+            :class:`ribs.archives.GridArchive`.
+        sigma (float or array-like): Standard deviation of the Gaussian
+            distribution. Note we assume the Gaussian is diagonal, so if this
+            argument is an array, it must be 1D.
+        lower_bounds (array-like): Upper bounds of the solution space. Passed in
+            by emitter
+        upper_bounds (array-like): Upper bounds of the solution space. Passed in
+            by emitter
+        seed (int): Value to seed the random number generator. Set to None to
+            avoid a fixed seed.
+      """
+
+    def __init__(self, iso_sigma, line_sigma, lower_bounds, upper_bounds, seed):
+
+        self._iso_sigma = iso_sigma
+        self._line_sigma = line_sigma
+        self._lower_bounds = lower_bounds
+        self._upper_bounds = upper_bounds
+
+        self._rng = np.random.default_rng(seed)
+
+    def ask(self, parents, directions):
```

Comment:
In this case, parents should be `(2, batch_size, solution_dim)` I believe. That way, we still use the same argument, but it just has different shapes. Add a note to the docstring regarding the shapes.